index.html

```diff
@@ -49,21 +55,29 @@ <h3>Some random information.</h3>
         </div>
         <div id="quoteDiv">
             <div id="quoteBox">
-                <p id="quote">This is an inspiring quote, or a testimonial from a customer. Maybe it's just filling up space, or maybe people will actually read it. Who knows? All I know is that it looks nice.</p>
+                <p id="quote">
+                    This is an inspiring quote, or a testimonial from a
+                    customer. Maybe it's just filling up space, or maybe people
+                    will actually read it. Who knows? All I know is that it
+                    looks nice.
+                </p>
                 <p id="quoter">-Thor, God of Thunder</p>
             </div>
         </div>
-        <div id="signupbanner">
-            <div id="signupdiv">
-                <div class="flexColumn">
+        <div class="sign-up-banner">
+            <div class="sign-up-banner__content">
+                <div>
                     <h4>Call to action! It's time!</h4>
-                    <p class="secondaryText">Sign up for our product by clicking on that button right over there!</p>
+                    <p class="secondaryText">
+                        Sign up for our product by clicking on that button right
+                        over there!
+                    </p>
                 </div>
-                <button class="signupbutton whiteoutline">Sign up</button>
+                <button type="button" class="button button-white">
+                    Sign up
+                </button>
             </div>
         </div>
-        <div id="copyright">
-            Copyright © The Odin Project 202!
-        </div>
+        <div id="copyright">Copyright &copy; The Odin Project 2024</div>
```

Comment:
Symbols like the copyright should be HTML encoded to prevent issues in certain browsers and character encoding formats.

https://mothereff.in/html-entities - turn on "allow named character references"

Not a big deal in practice, but a good habit nonetheless.